<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountConfigurationDialog</class>
 <widget class="QDialog" name="AccountConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add an account</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stacked_pages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_init">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Account parameters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Account name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edit_account_name"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>PGPKey </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_keys_list"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page__communities">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Communities membership</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListView" name="list_communities">
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="button_add_community">
              <property name="text">
               <string>Add a community</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_remove_community">
              <property name="text">
               <string>Remove selected community</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="button_previous">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_add_community</sender>
   <signal>clicked()</signal>
   <receiver>AccountConfigurationDialog</receiver>
   <slot>open_process_add_community()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_remove_community</sender>
   <signal>clicked()</signal>
   <receiver>AccountConfigurationDialog</receiver>
   <slot>action_remove_community()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_communities</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>AccountConfigurationDialog</receiver>
   <slot>open_process_edit_community(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_next</sender>
   <signal>clicked()</signal>
   <receiver>AccountConfigurationDialog</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_previous</sender>
   <signal>clicked()</signal>
   <receiver>AccountConfigurationDialog</receiver>
   <slot>previous()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_process_add_community()</slot>
  <slot>key_changed(int)</slot>
  <slot>action_remove_community()</slot>
  <slot>open_process_edit_community(QModelIndex)</slot>
  <slot>next()</slot>
  <slot>previous()</slot>
 </slots>
</ui>
